content/content-strategy/cms/cms-requirements/toc.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: nav/sections
+sections:
+  - Costs
+  - Hosting and security
+  - Content distribution network (CDN)
+  - CMS license
```

Comment:
Broken link. Does not match the heading in `content/content-strategy/cms/cms-requirements/heading4.md`

Reply:
Steve. Done.